lyluatex.lua

```diff
@@ -122,7 +132,7 @@ local function locate(file, includepaths)
         end
     end
     if not lfs.isfile(result) then result = kpse.find_file(file) end
-    if not result and file:sub(-3) ~= '.ly' then return locate(file..'.ly', includepaths) end
+    if not result and file:sub(-3) ~= ext then return locate(file..ext, includepaths) end
```

Comment:
This is not sufficient and will not even match the given `.xml` extention (only after changing to `sub(-4)`, so this must be more robust against different file extensions (changing to -4 will of course break compatibility with `.ly` files).

Reply:
Actually I'd say it should be checked if `file` does have an extension in the first place and then if it matches `ext`.

Reply:
It *is* checked.